lib/components/Widgets/WidgetContainer/index.tsx

```diff
@@ -16,23 +17,43 @@ export interface WidgetContainerProps {
     info?: string
     link?: { title: string; url: string }
   }
+  loading?: boolean
 }
 
 export const WidgetContainer = forwardRef<
   HTMLDivElement,
   WidgetContainerProps & { children: ReactNode }
->(({ header, children }, ref) => (
-  <Card ref={ref}>
+>(({ header, loading = false, children }, ref) => (
+  <Card ref={ref} aria-live="polite" aria-busy={loading}>
     <CardHeader>
-      <div className="flex grow flex-row items-center gap-1.5 truncate">
-        <CardTitle>{header.title}</CardTitle>
-        {header.subtitle && <CardSubtitle>{header.subtitle}</CardSubtitle>}
-        {header.info && <CardInfo content={header.info} />}
-      </div>
-      {header.link && (
-        <CardLink href={header.link.url} title={header.link.title} />
+      {loading ? (
+        <div className="flex h-6 w-full items-center" aria-hidden={true}>
+          <Skeleton className="h-4 w-full max-w-28" />
+        </div>
```

Comment:
Is the div wrapping the Skeleton really necessary? Also, instead of sending a className to the Skeleton maybe we want to simply change the looks in the Skeleton itself? Ideally the Skeleton will already know how to present itself and it won't need another component to tell it how to look like.

Reply:
@RamProg Good catch! The div is there to ensure the header maintains the same height as when there's a title. The skeleton bar is a bit less lower (drawing it at the same size as the text makes it looks a bit chunky, hah).

Regarding the Skeleton component, I'm quite taken with the freedom of passing classes. It's in line with shadcn's approach, after all. Maybe we could add some predefined styles to it.